ml/tests/dune

```diff
@@ -0,0 +1,6 @@
+(executable
+ (name e2e)
```

Comment:
What does `e2e` stand for?

Reply:
"End to end", because these are not unit test, it executes the thing entirely and check for any failure. It'd be nice to have better tests but that requires more work...